tests/objects/test_label_structure.py

```diff
@@ -1232,3 +1264,150 @@ def test_audio_object_can_be_added_edited_and_removed(ontology, empty_audio_labe
 
     label_row.remove_object(obj_instance)
     assert len(label_row.get_object_instances()) == 0
+
+
+def test_html_text_classification_can_be_added_edited_and_removed(ontology, empty_html_text_label_row: LabelRowV2):
+    label_row = empty_html_text_label_row
+    classification_instance = ClassificationInstance(checklist_classification, range_only=True)
+    classification_instance.set_for_frames(Range(start=0, end=1500))
+    range_list = classification_instance.range_list
+    assert len(range_list) == 1
+    assert range_list[0].start == 0
+    assert range_list[0].end == 1500
+
+    label_row.add_classification_instance(classification_instance)
+    assert len(label_row.get_classification_instances()) == 1
+    classification_instance.set_for_frames(Range(start=2000, end=2499))
+    range_list = classification_instance.range_list
```

Comment:
may just be personal:
This shouldn't be an ordered list imo, unless we are sorting it on requesting to ensure that we sort by the start index (If you are doing that then great!). The actual notion of classifications as you might view them in the label editor has the chronological ordering or here the lexicographic but if we are guaranteeing a stable order, it really needs to be derived from that.

Reply:
Sorry, I don't quite understand this. ChatGPT suggests that:

```
They are questioning whether range_list is inherently or intentionally ordered. If it's being treated as an ordered list, they are asking whether you are explicitly sorting it by a meaningful criterion (e.g., the start index of ranges).
```

If that is the case, then yes, `range_list` is being sorted by start index of ranges. See the `add_range` function in `encord/common/range_manager.py`.

Reply:
Cool, yeah I was asking whether it has a stable ordering